mautic/mautic
# Add check for PHP version support at install

This PR adds a check in the install for the user's PHP version and displays a recommendation to update if they are running a version which is no longer supported by PHP or is receiving security only updates.  To keep from needing to update this list too often, an array with all of Mautic's supported minor PHP branches is stored with the dates that each branch enters security support only and its end of support date; this will need to be updated roughly once a year as each new version is released.  http://php.net/supported-versions.php is used as the source data for active versions and the new releases were used to figure out the PHP 5.3 dates.

### How to Test

1) Apply patch
2) Run installer on PHP versions 5.3 thru 5.6.  PHP 5.3 should display a message saying the version is no longer supported, PHP 5.4 and 5.5 should display a message saying they are receiving security fixes only and nearing end of support, PHP 5.6 should show no message.
